scripts/plot_dashboard.py

```
@@ -0,0 +1,471 @@
+    ax_step.set_ylabel("Step response")
+    ax_step.set_xlabel("Sample")
+    for ax in (ax_imp, ax_step):
+        ax.grid(True, alpha=0.4)
+        ax.axhline(0.0, color="0.85", linewidth=0.5)
+
+    for dt in dtypes:
+        try:
+            ax_imp.plot(filt.process(impulse, dtype=dt), label=dt, linewidth=1.2)
+            ax_step.plot(filt.process(step, dtype=dt), label=dt, linewidth=1.2)
+        except Exception:
+            continue
```

Ruff (0.15.10)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)